app/albumShareRoom/[id]/page.tsx

```
@@ -0,0 +1,102 @@
+export default function AlbumShareRoomPage() {
+	const params = useParams();
+	const shareRoomId = String(params.id);
+	const searchParams = useSearchParams();
+	const sharedRoomTitle = searchParams.get("sharedRoomTitle");
+	const { currentUser } = useContext(authContext);
+	const [albums, setAlbums] = useState<Album[] | undefined>([]);
+	const [loading, setLoading] = useState(true);
+	const userId = currentUser?.uid;
+	const dispatch = useDispatch<AppDispatch>();
+
+	useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: shareRoomId

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)